OWNERS

```diff
@@ -5,11 +5,11 @@ approvers:
   - fededp
   - andreagit97
   - lucaguerra
-  - incertum
```

Comment:
As I step into my next professional chapter, I am officially transitioning out of my role as Falco maintainer. This decision has not been taken lightly, and while it's a farewell for now, I want to express my gratitude to this incredible team and community. Your dedication, collaboration, and passion have made this an unforgettable experience!

I'm thankful for everything we've built together, and I have no doubt the Falco community will continue to thrive. While I'm stepping away, I'll always be rooting for what comes next. Stay in touch :heart: !


```
   .:::.   .:::.
  :::::::.:::::::
  :::H4CK3R:::::  
   '::::::::::'
     '::::::'
       '::'
```

Ciao mΞl155@